juriscraper/opinions/united_states/state/wis.py

```diff
@@ -1,37 +1,116 @@
+            "begin_date": start,
+            "end_date": end,
+            "sortBy": "date",
+            "Submit": "Search",
+        }
+        self.url = f"{self.base_url}?{urlencode(params)}"
+
+    def _process_html(self) -> None:
+        """Process the HTML from wisconsin
+
+        :return: None
+        """
+        for row in self.html.xpath(".//table/tbody/tr"):
+            date, docket, caption, link = row.xpath("./td")
+            self.cases.append(
+                {
+                    "date": date.text,
+                    "name": caption.text,
+                    "url": urljoin(
+                        "https://www.wicourts.gov",
+                        link.xpath("./input")[0].name,
+                    ),
+                    "docket": docket.text,
+                }
+            )
+
+    def extract_from_text(self, scraped_text: str) -> dict:
+        """Extract citation from text
+
+        :param scraped_text: Text of scraped content
+        :return: date filed
+        """
+        first_line = scraped_text[:100].splitlines()[0]
+        match = re.search(self.cite_regex, first_line)
+
+        if match:
+            return {"Citation": {**match.groupdict(), "type": 8}}
+        return {}
+
+    def make_backscrape_iterable(self, kwargs: dict) -> None:
+        """Make backscrape itearble
+
+        Checks if backscrape start and end arguments have been passed
+        by caller, and parses them accordingly
 
-    def _get_case_names(self):
-        return get_table_column_text(self.html, 3, False, self.table_id)
+        :param kwargs: passed when initializing the scraper, may or
+            may not contain backscrape controlling arguments
+        :return None
+        """
+        start = kwargs.get("backscrape_start")
+        end = kwargs.get("backscrape_end")
 
-    def _get_download_urls(self):
-        path = f"{self.path_base}//td[4]//a[contains(., 'PDF')]/@href"
-        return self.html.xpath(path)
+        if start:
+            start = datetime.strptime(start, "%m/%d/%Y")
+        else:
+            start = self.first_opinion_date
+        if end:
+            end = datetime.strptime(end, "%m/%d/%Y")
+        else:
+            end = datetime.now()
 
-    def _get_case_dates(self):
-        path = f"{self.path_base}//td[1]"
-        cells = self.html.xpath(path)
-        return [convert_date_string(cell.text_content()) for cell in cells]
+        self.back_scrape_iterable = make_date_range_tuples(
+            start, end, self.days_interval
+        )
 
-    def _get_docket_numbers(self):
-        return get_table_column_text(self.html, 2, False, self.table_id)
+    def _download_backwards(self, dates: Tuple[date]) -> None:
+        """Set date range from backscraping args and scrape
 
-    def _get_precedential_statuses(self):
-        return ["Published"] * len(self.case_names)
+        :param dates: (start_date, end_date) tuple
+        :return None
+        """
+        self.set_url(*dates)
```

Comment:
It would be nice to add a `logger.info` call here for the dates that we are backscraping. Would make life easier for whoever runs the backscraper

Reply:
added